deploy/app/start-django-dev.sh

```diff
@@ -2,4 +2,4 @@
 
 wait-for-it "${PARROT_DB_HOST:-parrot-database}":5432 -s -t 180 \
 && python /app/src/manage.py migrate --noinput \
-&& python /app/src/manage.py runserver 0.0.0.0:8042
+&& python /app/src/manage.py runserver 127.0.0.1:8042
```

Comment:
This looks like it should be in a separate PR or a debug change that made its way into production code. :)

Reply:
Changed. Thanks!